hackerrank/a-very-big-sum/challenge.test.js

```diff
@@ -0,0 +1,18 @@
+const aVeryBigSum = require('./challenge');
+
+describe('sums a very large array of integers', () => {
+  test('adds 1000000001 + 1000000002 + 1000000003 + 1000000004 + 1000000005 to equal 5000000015', () => {
```

Comment:
Let's remove this and give a descriptive test description instead of hardcoded expectations values.